lib/assets/vega_lite/main.js

```diff
@@ -7,7 +7,85 @@ const DEFAULT_DATASET_NAME = "source_0";
 
 const throttledResize = throttle((view) => view.resize(), 1_000);
 
+const markColor = "#6583ff";
+const mainColors = ["#6583ff", "#ffa83f", "#e2474d", "#4aa148", "#2d4cdb", "#b7641f", "#bc1227", "#137518"];
+const blues = ["#b2c1ff", "#8ba2ff", "#6583ff", "#3E64ff", "#2d4cdb", "#1f37b7", "#132593"];
+const lightGray = "#e1e8f0";
+const mediumGray = "#445668";
+const darkGray = "#1c2a3a";
+const darkestGray = "#0d1829"
```

Comment:
Maybe name them with the gray scale we use, like `gray300`. Alternatively an object `const gray = { '300': ...}` :)